src/Skoruba.IdentityServer4.Admin/Helpers/StartupHelpers.cs

```diff
@@ -42,11 +42,41 @@
 using Skoruba.IdentityServer4.Admin.EntityFramework.SqlServer.Extensions;
 using Skoruba.IdentityServer4.Admin.EntityFramework.PostgreSQL.Extensions;
 using Skoruba.IdentityServer4.Admin.EntityFramework.Helpers;
+using Skoruba.IdentityServer4.Admin.Helpers.Messaging.Interfaces;
+using SendGrid;
+using Skoruba.IdentityServer4.Admin.Helpers.Messaging;
 
 namespace Skoruba.IdentityServer4.Admin.Helpers
 {
     public static class StartupHelpers
     {
+        /// <summary>
```

Comment:
This function wires up the injection of the email services.  It also exists in the STS.Identity project, and was generic enough to be considered implementing in an extension within a "shared project" between both Admin and STS.Identity.